models/InvoiceItem.php

```diff
@@ -60,7 +60,7 @@ public function calculateTotals()
             $this->tax = $taxClass->getTotalTax($this->subtotal);
         }
 
-        $this->total = $this->subtotal + $this->tax;
+        $this->total = $this->subtotal;//+ $this->tax;
```

Comment:
Why?

Reply:
Sorry, I pushed some test code on the wrong branch.